products/azure_rm_batchaccount/api.yaml

```diff
@@ -0,0 +1,101 @@
+--- !ruby/object:Api::Product
+name: Azure BatchAccount Management
+prefix: azure_rm_batchaccount
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: NotUsedInAzure
+scopes:
+  - NotUsedInAzure
+azure_namespace: 'Microsoft.Batch'
+azure_version: '2018-12-01'
+objects:
+  - !ruby/object:Api::Resource
+    name: BatchAccount
+    api_name: Batch
+    base_url: NotUsedInAzure
+
+    azure_sdk_definition: !ruby/object:Api::Azure::SDKDefinition
+      provider_name: batch
+      read: !ruby/object:Api::Azure::SDKOperationDefinition
+        go_func_name: Get
+        request:
+          'resourceGroupName': !ruby/object:Api::Azure::SDKTypeDefinition::StringObject
+            id_portion: resourceGroupName
+            go_variable_name: resourceGroupName
+          'accountName': !ruby/object:Api::Azure::SDKTypeDefinition::StringObject
+            id_portion: accountName
+            go_variable_name: accountName
+        response:
+      delete: !ruby/object:Api::Azure::SDKOperationDefinition

```

Comment:
Please declare that the `delete` operation is "Async" before `go_func_name`. You may infer whether the operation is async or not by looking for the [`x-ms-long-running-operation`](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/batch/resource-manager/Microsoft.Batch/stable/2018-12-01/BatchManagement.json#L208) property in api-spec.

```yaml
    async: true
```

Reply:
done